client/src/app/rootReducer.ts

```diff
@@ -1,10 +1,10 @@
 import { combineReducers } from '@reduxjs/toolkit';
 import authReducer from '../features/auth/authSlice';
```

Comment:
why `authSlice`? It's called clusters, as i know. So better `import authReducer from '../features/auth/authCluster';`